### PR TITLE
Updated `ember-frost-core@^1.0.0`

### PR DESCRIPTION
**Pull request by dafortin**
### This project uses [semver](semver.org), please check the scope of this pr:
 - [ ] #patch# - backwards-compatible bug fix
 - [ ] #minor# - adding functionality in a backwards-compatible manner
 - [x] #major# - incompatible API change

# CHANGELOG
Updated `ember-frost-core` to `^1.0.0`

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -5,12 +5,12 @@
...
-    "highlightjs": "^9.2.0",
+    "highlightjs": "^9.8.0",
     "lodash": "^4.10.0",
-    "marked": "^0.3.5",
-    "perfect-scrollbar": ">=0.6.7 <2.0.0",
+    "marked": "^0.3.6",
     "sinonjs": "1.17.1",
-    "node-uuid": "1.4.7"
+    "node-uuid": "1.4.7",
+    "ember-mocha-adapter": "~0.3.1"
```
Comment (rox163) [inline bower.json]: alphabetical :)

Comment (dafortin) [reply bower.json]: sorry :P

**Comment by rox163** [comment]
:+1:

[Approved with PullApprove](https://pullapprove.com/ciena-frost/ember-frost-sort/pull-request/44/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=ciena-frost/ember-frost-sort)

**Comment by oadeojo** [comment]
👍 

[Approved with PullApprove](https://pullapprove.com/ciena-frost/ember-frost-sort/pull-request/44/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=ciena-frost/ember-frost-sort)

File: bower.json [hunk 2]
```diff
@@ -5,10 +5,10 @@
     "ember-cli-shims": "0.1.1",
     "ember-inflector": "1.3.1",
     "ember-mocha": "~0.8.11",
-    "highlightjs": "^9.2.0",
+    "ember-mocha-adapter": "~0.3.1",
+    "highlightjs": "^9.8.0",
     "lodash": "^4.10.0",
-    "marked": "^0.3.5",
-    "perfect-scrollbar": ">=0.6.7 <2.0.0",
```
Comment (rox163) [inline bower.json:11]: oh this mightve had some setup in ember-cli-build or index.js

**Pull request merged**